github action deploy エラーの修正

[
  {
    "textPayload": "Starting new instance. Reason: DEPLOYMENT_ROLLOUT - Instance started due to traffic shifting between revisions due to deployment, traffic split adjustment, or deployment health check.",
    "insertId": "69b94cb8000569902e50ea90",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "service_name": "sys-dev-automation",
        "configuration_name": "sys-dev-automation",
        "location": "asia-northeast1"
      }
    },
    "timestamp": "2026-03-17T12:44:40.354704Z",
    "severity": "INFO",
    "labels": {
      "scaling_reason": "DEPLOYMENT_ROLLOUT",
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fvarlog%2Fsystem",
    "receiveTimestamp": "2026-03-17T12:44:40.360433995Z"
  },
  {
    "textPayload": "> app@1.0.0 start",
    "insertId": "69b94cb900035c37ae995dda",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "configuration_name": "sys-dev-automation",
        "service_name": "sys-dev-automation",
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "location": "asia-northeast1"
      }
    },
    "timestamp": "2026-03-17T12:44:41.220215Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstdout",
    "receiveTimestamp": "2026-03-17T12:44:41.226798086Z"
  },
  {
    "textPayload": "> node dist/src/index.js",
    "insertId": "69b94cb900035c49a8a2dc75",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "configuration_name": "sys-dev-automation",
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "service_name": "sys-dev-automation",
        "location": "asia-northeast1"
      }
    },
    "timestamp": "2026-03-17T12:44:41.220233Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstdout",
    "receiveTimestamp": "2026-03-17T12:44:41.226798086Z"
  },
  {
    "textPayload": "node:internal/modules/cjs/loader:1386",
    "insertId": "69b94cb90003ff9c4f4760ec",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "location": "asia-northeast1",
        "service_name": "sys-dev-automation",
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "configuration_name": "sys-dev-automation"
      }
    },
    "timestamp": "2026-03-17T12:44:41.262044Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstderr",
    "receiveTimestamp": "2026-03-17T12:44:41.267123674Z"
  },
  {
    "textPayload": "  throw err;",
    "insertId": "69b94cb90003ffb0300a8fd2",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "location": "asia-northeast1",
        "configuration_name": "sys-dev-automation",
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "service_name": "sys-dev-automation"
      }
    },
    "timestamp": "2026-03-17T12:44:41.262064Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstderr",
    "receiveTimestamp": "2026-03-17T12:44:41.267123674Z"
  },
  {
    "textPayload": "  ^",
    "insertId": "69b94cb90003ffb5b744837e",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "location": "asia-northeast1",
        "configuration_name": "sys-dev-automation",
        "revision_name": "sys-dev-automation-00004-tsp",
        "project_id": "se-auto-agent-20251222",
        "service_name": "sys-dev-automation"
      }
    },
    "timestamp": "2026-03-17T12:44:41.262069Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstderr",
    "receiveTimestamp": "2026-03-17T12:44:41.267123674Z"
  },
  {
    "textPayload": "Error: Cannot find module '/app/dist/src/index.js'\n    at Function._resolveFilename (node:internal/modules/cjs/loader:1383:15)\n    at defaultResolveImpl (node:internal/modules/cjs/loader:1025:19)\n    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1030:22)\n    at Function._load (node:internal/modules/cjs/loader:1192:37)\n    at TracingChannel.traceSync (node:diagnostics_channel:328:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:237:24)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:171:5)\n    at node:internal/main/run_main_module:36:49 {",
    "insertId": "69b94cb90003ffd505e4b5b3",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "service_name": "sys-dev-automation",
        "revision_name": "sys-dev-automation-00004-tsp",
        "project_id": "se-auto-agent-20251222",
        "configuration_name": "sys-dev-automation",
        "location": "asia-northeast1"
      }
    },
    "timestamp": "2026-03-17T12:44:41.262101Z",
    "severity": "ERROR",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstderr",
    "receiveTimestamp": "2026-03-17T12:44:41.267123674Z",
    "errorGroups": [
      {
        "id": "CNj546-vjOW8pAE"
      }
    ]
  },
  {
    "textPayload": "  code: 'MODULE_NOT_FOUND',",
    "insertId": "69b94cb90003ffd8a2ba1292",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "service_name": "sys-dev-automation",
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "configuration_name": "sys-dev-automation",
        "location": "asia-northeast1"
      }
    },
    "timestamp": "2026-03-17T12:44:41.262104Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstderr",
    "receiveTimestamp": "2026-03-17T12:44:41.267123674Z"
  },
  {
    "textPayload": "  requireStack: []",
    "insertId": "69b94cb90003ffdb23d6acdc",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "location": "asia-northeast1",
        "service_name": "sys-dev-automation",
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "configuration_name": "sys-dev-automation"
      }
    },
    "timestamp": "2026-03-17T12:44:41.262107Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstderr",
    "receiveTimestamp": "2026-03-17T12:44:41.267123674Z"
  },
  {
    "textPayload": "}",
    "insertId": "69b94cb90003ffde7a0020e2",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "location": "asia-northeast1",
        "configuration_name": "sys-dev-automation",
        "revision_name": "sys-dev-automation-00004-tsp",
        "project_id": "se-auto-agent-20251222",
        "service_name": "sys-dev-automation"
      }
    },
    "timestamp": "2026-03-17T12:44:41.262110Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstderr",
    "receiveTimestamp": "2026-03-17T12:44:41.267123674Z"
  },
  {
    "textPayload": "Node.js v22.22.1",
    "insertId": "69b94cb90003ffe28deffe02",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "service_name": "sys-dev-automation",
        "configuration_name": "sys-dev-automation",
        "location": "asia-northeast1"
      }
    },
    "timestamp": "2026-03-17T12:44:41.262114Z",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fstderr",
    "receiveTimestamp": "2026-03-17T12:44:41.267123674Z"
  },
  {
    "textPayload": "Container called exit(1).",
    "insertId": "69b94cb90009fe7343768559",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "service_name": "sys-dev-automation",
        "configuration_name": "sys-dev-automation",
        "location": "asia-northeast1"
      }
    },
    "timestamp": "2026-03-17T12:44:41.654931045Z",
    "severity": "WARNING",
    "labels": {
      "container_name": "sys-dev-automation-1",
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fvarlog%2Fsystem",
    "receiveTimestamp": "2026-03-17T12:44:41.690944215Z"
  },
  {
    "textPayload": "Default STARTUP TCP probe failed 1 time consecutively for container \"sys-dev-automation-1\" on port 8080. The instance was not started.\nConnection failed with status CANCELLED.",
    "insertId": "69b94cb9000b774c83f615eb",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "location": "asia-northeast1",
        "revision_name": "sys-dev-automation-00004-tsp",
        "project_id": "se-auto-agent-20251222",
        "service_name": "sys-dev-automation",
        "configuration_name": "sys-dev-automation"
      }
    },
    "timestamp": "2026-03-17T12:44:41.751436Z",
    "severity": "ERROR",
    "labels": {
      "instanceId": "00da6cd2c4c4e384ce2f041ff1eb3e1bdfdeb47a9d2ba8cd5cbc947460314ac899e9a365589b337c24b1683b9edeb53145bec0e79e997f93dd306edb1efe69a335d9d0e12fc67f5607a4475610ec5a993dfa"
    },
    "logName": "projects/se-auto-agent-20251222/logs/run.googleapis.com%2Fvarlog%2Fsystem",
    "receiveTimestamp": "2026-03-17T12:44:42.025065740Z"
  },
  {
    "protoPayload": {
      "@type": "type.googleapis.com/google.cloud.audit.AuditLog",
      "status": {
        "code": 9,
        "message": "Ready condition status changed to False for Revision sys-dev-automation-00004-tsp with message: The user-provided container failed to start and listen on the port defined provided by the PORT=8080 environment variable within the allocated timeout. This can happen when the container port is misconfigured or if the timeout is too short. The health check timeout can be extended. Logs for this revision might contain more information.\n\nLogs URL: https://console.cloud.google.com/logs/viewer?project=se-auto-agent-20251222&resource=cloud_run_revision/service_name/sys-dev-automation/revision_name/sys-dev-automation-00004-tsp&advancedFilter=resource.type%3D%22cloud_run_revision%22%0Aresource.labels.service_name%3D%22sys-dev-automation%22%0Aresource.labels.revision_name%3D%22sys-dev-automation-00004-tsp%22 \nFor more troubleshooting guidance, see https://cloud.google.com/run/docs/troubleshooting#container-failed-to-start"
      },
      "serviceName": "run.googleapis.com",
      "methodName": "/Services.ReplaceService",
      "resourceName": "namespaces/se-auto-agent-20251222/revisions/sys-dev-automation-00004-tsp",
      "response": {
        "metadata": {
          "name": "sys-dev-automation-00004-tsp",
          "namespace": "321300615156",
          "selfLink": "/apis/serving.knative.dev/v1/namespaces/321300615156/revisions/sys-dev-automation-00004-tsp",
          "uid": "d93a1db3-7167-4337-8779-37bef3d80acc",
          "resourceVersion": "AAZNN7L9XVY",
          "generation": 1,
          "creationTimestamp": "2026-03-17T12:44:34.533407Z",
          "labels": {
            "client.knative.dev/nonce": "rgscyqtgbm",
            "serving.knative.dev/configuration": "sys-dev-automation",
            "serving.knative.dev/configurationGeneration": "4",
            "serving.knative.dev/service": "sys-dev-automation",
            "serving.knative.dev/serviceUid": "326ae882-8a90-4d90-9b6a-1cb7285c6a5d",
            "serving.knative.dev/route": "sys-dev-automation",
            "cloud.googleapis.com/location": "asia-northeast1",
            "run.googleapis.com/startupProbeType": "Default"
          },
          "annotations": {
            "autoscaling.knative.dev/maxScale": "20",
            "run.googleapis.com/client-name": "gcloud",
            "run.googleapis.com/client-version": "557.0.0",
            "serving.knative.dev/creator": "github-actions-deployer@se-auto-agent-20251222.iam.gserviceaccount.com",
            "run.googleapis.com/build-source-location": "{\"sys-dev-automation-1\":\"gs://run-sources-se-auto-agent-20251222-asia-northeast1/services/sys-dev-automation/1773751370.402112-aee6dc102b374c73816fa8c89036b3ea.zip#1773751370794563\"}",
            "run.googleapis.com/operation-id": "3bf3cfff-4c05-4dfd-9011-e02f258b0026",
            "run.googleapis.com/startup-cpu-boost": "true"
          },
          "ownerReferences": [
            {
              "kind": "Configuration",
              "name": "sys-dev-automation",
              "uid": "c12cda7b-5f65-40cf-a177-acf0b5dd57dd",
              "apiVersion": "serving.knative.dev/v1",
              "controller": true,
              "blockOwnerDeletion": true
            }
          ]
        },
        "apiVersion": "serving.knative.dev/v1",
        "kind": "Revision",
        "spec": {
          "containerConcurrency": 80,
          "timeoutSeconds": 300,
          "serviceAccountName": "321300615156-compute@developer.gserviceaccount.com",
          "containers": [
            {
              "name": "sys-dev-automation-1",
              "image": "asia-northeast1-docker.pkg.dev/se-auto-agent-20251222/cloud-run-source-deploy/sys-dev-automation@sha256:c75efc5792188fa22947b5c01055d990dad4928cdd40076c2f185fa165b250d4",
              "ports": [
                {
                  "name": "http1",
                  "containerPort": 8080
                }
              ],
              "env": [
                {
                  "name": "SLACK_BOT_TOKEN",
                  "valueFrom": {
                    "secretKeyRef": {
                      "key": "latest",
                      "name": "SLACK_BOT_TOKEN"
                    }
                  }
                },
                {
                  "name": "SLACK_SIGNING_SECRET",
                  "valueFrom": {
                    "secretKeyRef": {
                      "key": "latest",
                      "name": "SLACK_SIGNING_SECRET"
                    }
                  }
                },
                {
                  "name": "GITHUB_TOKEN",
                  "valueFrom": {
                    "secretKeyRef": {
                      "key": "latest",
                      "name": "GITHUB_TOKEN"
                    }
                  }
                },
                {
                  "name": "GITHUB_REPO",
                  "valueFrom": {
                    "secretKeyRef": {
                      "key": "latest",
                      "name": "GITHUB_REPO"
                    }
                  }
                },
                {
                  "name": "GEMINI_API_KEY",
                  "valueFrom": {
                    "secretKeyRef": {
                      "key": "latest",
                      "name": "GEMINI_API_KEY"
                    }
                  }
                },
                {
                  "name": "USE_FIRESTORE",
                  "value": "true"
                },
                {
                  "name": "SLACK_SOCKET_MODE",
                  "value": "false"
                }
              ],
              "resources": {
                "limits": {
                  "cpu": "1000m",
                  "memory": "1Gi"
                }
              },
              "startupProbe": {
                "timeoutSeconds": 240,
                "periodSeconds": 240,
                "failureThreshold": 1,
                "tcpSocket": {
                  "port": 8080
                }
              }
            }
          ]
        },
        "status": {
          "observedGeneration": 1,
          "conditions": [
            {
              "type": "Ready",
              "status": "False",
              "reason": "HealthCheckContainerError",
              "message": "The user-provided container failed to start and listen on the port defined provided by the PORT=8080 environment variable within the allocated timeout. This can happen when the container port is misconfigured or if the timeout is too short. The health check timeout can be extended. Logs for this revision might contain more information.\n\nLogs URL: https://console.cloud.google.com/logs/viewer?project=se-auto-agent-20251222&resource=cloud_run_revision/service_name/sys-dev-automation/revision_name/sys-dev-automation-00004-tsp&advancedFilter=resource.type%3D%22cloud_run_revision%22%0Aresource.labels.service_name%3D%22sys-dev-automation%22%0Aresource.labels.revision_name%3D%22sys-dev-automation-00004-tsp%22 \nFor more troubleshooting guidance, see https://cloud.google.com/run/docs/troubleshooting#container-failed-to-start",
              "lastTransitionTime": "2026-03-17T12:44:41.752918Z"
            },
            {
              "type": "ContainerHealthy",
              "status": "False",
              "reason": "HealthCheckContainerError",
              "message": "The user-provided container failed to start and listen on the port defined provided by the PORT=8080 environment variable within the allocated timeout. This can happen when the container port is misconfigured or if the timeout is too short. The health check timeout can be extended. Logs for this revision might contain more information.\n\nLogs URL: https://console.cloud.google.com/logs/viewer?project=se-auto-agent-20251222&resource=cloud_run_revision/service_name/sys-dev-automation/revision_name/sys-dev-automation-00004-tsp&advancedFilter=resource.type%3D%22cloud_run_revision%22%0Aresource.labels.service_name%3D%22sys-dev-automation%22%0Aresource.labels.revision_name%3D%22sys-dev-automation-00004-tsp%22 \nFor more troubleshooting guidance, see https://cloud.google.com/run/docs/troubleshooting#container-failed-to-start",
              "lastTransitionTime": "2026-03-17T12:44:41.752918Z"
            },
            {
              "type": "ContainerReady",
              "status": "True",
              "message": "Container image import completed in 3.55s.",
              "lastTransitionTime": "2026-03-17T12:44:38.602296Z"
            },
            {
              "type": "ResourcesAvailable",
              "status": "True",
              "message": "Provisioning imported containers completed in 1.62s. Checking container health. This will wait for up to 4m for the configured startup probe, including an initial delay of 0s.",
              "lastTransitionTime": "2026-03-17T12:44:40.227764Z"
            },
            {
              "type": "Retry",
              "status": "True",
              "reason": "ImmediateRetry",
              "message": "System will retry after 00:00 from lastTransitionTime for attempt 0.",
              "lastTransitionTime": "2026-03-17T12:44:40.227764Z",
              "severity": "Info"
            }
          ],
          "logUrl": "https://console.cloud.google.com/logs/viewer?project=se-auto-agent-20251222&resource=cloud_run_revision/service_name/sys-dev-automation/revision_name/sys-dev-automation-00004-tsp&advancedFilter=resource.type%3D%22cloud_run_revision%22%0Aresource.labels.service_name%3D%22sys-dev-automation%22%0Aresource.labels.revision_name%3D%22sys-dev-automation-00004-tsp%22",
          "imageDigest": "asia-northeast1-docker.pkg.dev/se-auto-agent-20251222/cloud-run-source-deploy/sys-dev-automation@sha256:c75efc5792188fa22947b5c01055d990dad4928cdd40076c2f185fa165b250d4",
          "containerStatuses": [
            {
              "name": "sys-dev-automation-1",
              "imageDigest": "asia-northeast1-docker.pkg.dev/se-auto-agent-20251222/cloud-run-source-deploy/sys-dev-automation@sha256:c75efc5792188fa22947b5c01055d990dad4928cdd40076c2f185fa165b250d4"
            }
          ]
        },
        "@type": "type.googleapis.com/google.cloud.run.v1.Revision"
      }
    },
    "insertId": "z0gfwrdark4",
    "resource": {
      "type": "cloud_run_revision",
      "labels": {
        "service_name": "sys-dev-automation",
        "project_id": "se-auto-agent-20251222",
        "revision_name": "sys-dev-automation-00004-tsp",
        "configuration_name": "sys-dev-automation",
        "location": "asia-northeast1"
      }
    },
    "timestamp": "2026-03-17T12:44:41.762053Z",
    "severity": "ERROR",
    "logName": "projects/se-auto-agent-20251222/logs/cloudaudit.googleapis.com%2Fsystem_event",
    "receiveTimestamp": "2026-03-17T12:44:42.483870991Z"
  }
]

--- a/.dockerignore
+++ b/.dockerignore
@@ -8,4 +8,3 @@ npm-debug.log
 .dockerignore
 README.md
 AGENTS.md
-package-lock.json

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -7,6 +7,7 @@
     "forceConsistentCasingInFileNames": true,
     "strict": true,
     "skipLibCheck": true,
+    "rootDir": ".",
     "outDir": "./dist"
   },
   "include": ["src/**/*", "tests/**/*"]

